eth-types/src/lib.rs

```diff
@@ -21,7 +21,7 @@ pub mod bytecode;
 pub mod evm_types;
 pub mod geth_types;
 
-pub use bytecode::Bytecode;
+pub use bytecode::{Bytecode, EMPTY_HASH, EMPTY_HASH_LE};
```

Comment:
Are we sure we want to define `EMPTY_HASH` inside `bytecode` crate?
Why not define it in keccak and import it from there?